docker/Dockerfile.agent-data-plane

```diff
@@ -78,7 +82,7 @@ USER root
 RUN test -d /usr/share/ca-certificates || (apt-get update && \
     apt-get install -y --no-install-recommends ca-certificates=20240203~22.04.1 && \
     apt-get clean && rm -rf /var/lib/apt/lists)
-COPY --from=builder /adp/target/${BUILD_PROFILE}/agent-data-plane /usr/local/bin/agent-data-plane
+COPY --from=builder /adp/target/bin/agent-data-plane /usr/local/bin/agent-data-plane
```

Comment:
Why are we still modifying the ADP Dockerfile? 😅

Reply:
Ohh, that was done previously to allow us not to rely on the BUILD_PROFILE on the last build step. I can remove it if you prefer. The final build is not affected nor includes the Python interpreter

Reply:
Yeah, I'd rather just leave the file as-is if we don't _need_ to change it.